validator/lib/validateBoundingVolume.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+var Promise = require('bluebird');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+module.exports = validateVolume;
+
+function validateVolume(tileset) {
+    return new Promise(function(resolve) {
+        validateNode(tileset.root, tileset, resolve);
+    });
+}
+
+var scratchCartographic = new Cartographic();
+var scratchRect = new Rectangle();
+var scratchTileRect = new Rectangle();
+
+function regionInsideRegion() {
+    return (Rectangle.contains(scratchTileRect, Cartographic.fromRadians(scratchRect.west,
+        scratchRect.north, 0.0, scratchCartographic))
+    && Rectangle.contains(scratchTileRect, Cartographic.fromRadians(scratchRect.west,
+        scratchRect.south, 0.0, scratchCartographic))
+    && Rectangle.contains(scratchTileRect, Cartographic.fromRadians(scratchRect.east,
+        scratchRect.north, 0.0, scratchCartographic))
+    && Rectangle.contains(scratchTileRect, Cartographic.fromRadians(scratchRect.east,
+        scratchRect.south, 0.0, scratchCartographic)));
```

Comment:
Just noticed.. for these you can use `Rectangle.southwest` and similar functions.